docs-src/ref-programs.scrbl

```diff
@@ -1902,7 +1921,26 @@ expression. For example:
     .invariant(...)
     .while( keepGoing() )
     .case(...)
-    .timeout( timeRemaining(), ...) }
+    .timeout( timeRemaining(), () => { ... }) }
+
+When dealing with absolute deadlines in @reachin{parallel_reduce}, there is a common pattern in the
```

Comment:
I think that the expansion should be explained only once and instead this block of docs should say... "this pattern is so common, it is abbreviate as `.time_remaining`", which would be a link to the further explanation. I am scared of copy & paste